javascript/browser-apis/reading-writing/blobs.md

```diff
@@ -33,34 +20,44 @@ links:
 ---
 ## Content
 
-A *blob* object represents an "almost file" object of raw data. It represents data that isn't in JavaScript's standard format. You use *FileReader* to read data from a blob.
+A binary large object, or *blob*, represents a file-like object of immutable raw data. It represents data that isn't in JavaScript's standard format. You use `FileReader` to read data from a blob.
 
 To create a blob, use the blob constructor:
 ```javascript
-var debug = {hello: "world"};
-var blob = new Blob([
-  JSON.stringify(debug, null, 2)
-], {type : 'application/json'});
+let debug = {
+  hello: "world",
+};
+
+let blob = new Blob(
+  [JSON.stringify(debug, null, 2)],
+  {
+    type: "application/json",
+  }
+);
 ```
 
-To create a subset of a blob, use the *slice()* method:
-```javascript
-var blob = instanceOfBlob.slice([
-  start [, end [, contentType]]
-]);
-```
+A blob can be very large, so this means it can contain audio and video data. They can be created dynamically, and used as files when coupled with blobl URLs.
+
+Although blobs aren't particularly useful when on their own, their redeeming feature is being able to be used in many calls that are meant for `File`s.
+
+For example, a `Blob` could represent something like a temporary file, which can then be linked to and treated as being a file on a web server.
```

Comment:
I'd drop the "something" -> although the term is used in common language it strikes me as vague in this context

Reply:
so the main advantage of blobs is that they can be used as in-memory files? maybe emphasise on this point more, if there aren't other benefits